Casks/git-credential-manager-core.rb

```diff
@@ -2,8 +2,8 @@
   name 'Git Credential Manager Core'
   homepage 'https://aka.ms/gcmcore'
 
-  version "2.0.435.9025"
-  sha256 'a24e79a2ed00fb5ac3a6052c2a69d7a571455f423b26b8c1e0abc88bfbdc1e4b'
+  version "2.0.452.3248"
+  sha256 '8026fa9589a41d5a7b56e17f7001f84053d05eab03d0381097d1696dca43ae78'
 
   url "https://github.com/microsoft/Git-Credential-Manager-Core/releases/download/v#{version.major_minor_patch}-beta/gcmcore-osx-#{version}.pkg"
```

Comment:
@mjcheetham: this `url` still has `-beta` and needs to be changed in the workflow in `microsoft/git-credential-manager-core`.